modules/vf-graphql-holochain/types.ts

```diff
@@ -136,20 +136,27 @@ export enum VfModule {
 // default 'full suite' VF module set supported by Holo-REA
 
 export const DEFAULT_VF_MODULES = [
-  VfModule.Agent,
-  VfModule.Agreement,
+  // Specification DNA
```

Comment:
I've been considering these groupings of zomes into DNAs "the default suite", but my hope is that eventually the particular composition will be meaningless and that 3rd-party app developers will have no trouble creating their own arrangements of zomes and indexes without too much trouble.

Reply:
Agreed!

Reply:
DevHub actually is a really good tool for mixing and matching.

Reply:
DNAs? Or can it do per-zome remixing as well?

Reply:
Per zome as well.